src/poppler/document.py

```diff
@@ -277,18 +280,22 @@ def unlock(self, owner_password, user_password):
         return self._document.unlock(owner_password, user_password)
 
 
-def load_from_file(file_name, owner_password=None, user_password=None):
+def load_from_file(
+    file_name: str, owner_password: str = "", user_password: str = ""
+) -> Document:
+    if not isinstance(file_name, str):
+        raise TypeError("file_name MUST be str")
     return Document(
-        document.load_from_file(
-            str(file_name), owner_password or "", user_password or ""
-        )
+        document.load_from_file(file_name, owner_password, user_password)
     )
 
 
-def load_from_data(file_data: bytes, owner_password=None, user_password=None):
+def load_from_data(
+    file_data: bytes, owner_password: str = "", user_password: str = ""
```

Comment:
I prefer using None for default It is more explicit. With you suggestion, it is no longer possible to use None as argument to the function.

Reply:
Apart from that, *usually* using None means no password, while "" could be an empty one. Don't know, how poppler/pdf deals with empty passwords...

Reply:
I don't know much about PDFs but if empty password is a valid password for a PDF then I agree with you being `None` for default. otherwise it is better to use empty string as default.
Waiting for your comment @cbrunet

Reply:
The C++ API expects an empty string. However, semantically, in Python, `None` means no value. Since we are in the context of a library, I prefer to let the user choose the value that is more suitable for him. Furthermore, annotating the password field as `Optional[str]` is more expressive, imho.